### DISCUSSION TITLE
ios-intercept-crypto no results on frida 14.2.14

### DISCUSSION THREAD

**Discussion by masbog** [noobpk/frida-ios-hook#7] [Q&A]
Hi, 

wanna to know that script frida-scripts/ios-intercept-crypto*.js was not worked on my 14.2.14 version on iOS, i don't know why? it should downgrade the frida version first? on my computer and my iOS Device?

Thanks

**Comment by yazgx97** [comment]
Hi,
The Frida is currently having some issues with new iOS versions 14.x . You can view the ios 14.x issues on [github frida](https://github.com/frida/frida/issues) and wait for new updates.

**Reply by masbog** [reply]
All right, it seem frida was updated and got issues on ios14, maybe will try on lower ios version first 😞

**Reply by yazgx97** [reply]
Yes, If you can downgrade iOS version 13. Everything will work fine :D :D
